examples/flash.rs

```diff
@@ -0,0 +1,63 @@
+//! Example of Flash erasing, writing and reading with the stm32h743zi (rm0433)
+//! Assumes this example runs at sector 0 of bank 1 (0x0800_0000)
+
+#![deny(warnings)]
+#![no_main]
+#![no_std]
+
+use cortex_m_rt::entry;
+#[macro_use]
+mod utilities;
+use log::info;
+use stm32h7xx_hal::{flash::Bank, pac, prelude::*};
+
+#[entry]
+fn main() -> ! {
+    info!("stm32h7xx-hal example - erase, write and read FLASH");
```

Comment:
Add a call to init here
```suggestion
    utilities::logger::init();
    info!("stm32h7xx-hal example - erase, write and read FLASH");
```